packages/harden/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
```

Comment:
What is the scope of "project"? Do you mean "package"?

Reply:
Yes. I’ll fix uniformly in follow-up. This boilerplate is in every package.